src/lipyphilic/analysis/memb_thickness.py

```diff
@@ -169,7 +169,8 @@ def __init__(self, universe, lipid_sel, leaflets, n_bins=1, interpolate=False, r
             of higher-resolution grids. The default is False.
         return_surface : bool, optional
             If True, the height of the bilayer at grid point at each frame is returned as
-            numpy ndarray. The default is False.
+            numpy ndarray. The 2D grid will be stored in self.memb_thickness_grid. The
+            default is False.
```

Comment:
Thanks for spotting this. I know it wasn't added in this PR, but could you store the result in `self.results.memb_thickness_grid` instead, and add a property `self.memb_thickness_grid` that returns this (just so that we're using the MDAnalysis `Results` class whilst also not breaking existing code)